Dragusev1ch/BankingSolution
# Release v1.0.0

This pool-request completes the work on the current version of the project, combining all the changes from the develop branch into the main one.

Main changes:

- Core functionality implemented:
- Automated tests have been added to check the correctness of the functionality.
- API documentation has been customized using Swagger/OpenAPI.
- Provided logging of all API actions for easy tracking of operations.

Additional changes:

- Bug fixes
- Code refactoring

Verification instructions:

Read the documentation in README.md.
Test the basic functionality using the Swagger UI (/swagger).
Run tests using dotnet test to make sure the implementation is correct.
